bedrock/careers/templates/careers/base.html

```diff
@@ -0,0 +1,76 @@
+{% set default_slug = "Mozilla Careers &mdash; Feel good about your work again" %}
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}" class="no-js"
+      data-ga-code="{{ settings.GA_ACCOUNT_CODE }}"
+      data-gtm-code="{{ settings.GTM_ACCOUNT_CODE }}">
+  <head>
+    {# meta #}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta property="og:description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <meta property="og:image" content="https://careers.mozilla.org{{ meta_image|default(static('img/careers/moz-m.png'), true)|safe }}">
+    <meta property="og:image:alt" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta name="twitter:title" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta name="twitter:description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <meta name="twitter:image" content="https://careers.mozilla.org{{ meta_image|default(static('img/careers/moz-m.png'), true)|safe }}">
+    <meta name="twitter:card" content="summary">
+    <meta name="description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <title>{{ meta_title|default(default_slug, true)|safe }}</title>
+
+    {# css #}
+    {% block site_css %}
+      {{ css_bundle('careers') }}
+    {% endblock %}
+
+    {# js #}
+    <script src="{{ static('js/careers/dnt-helper.js') }}"></script>
```

Comment:
This can probably be replaced directly with the copy that's already in bedrock
```suggestion
    <script src="{{ static('js/base/dnt-helper.js') }}"></script>
```